app/models/user.rb

```diff
@@ -553,23 +551,15 @@ def admin?
   end
 
   def delegate_roles
-    active_roles.filter { |role| UserRole.is_group_type?(role, UserGroup.group_types[:delegate_regions]) }
+    active_roles(include_converted: false).filter { |role| UserRole.is_group_type?(role, UserGroup.group_types[:delegate_regions]) }
   end
 
   def any_kind_of_delegate?
-    active_roles.any? { |role| UserRole.is_group_type?(role, UserGroup.group_types[:delegate_regions]) }
+    !delegate_roles.empty
```

Comment:
Attention: This must be `empty?` with a question mark at the end

Reply:
`delegate_roles.any?` looks better. Thanks for pointing `empty` vs `empty?`.